src/authors-details.js

```diff
@@ -0,0 +1,29 @@
+//TODO: data related to each author on the blog posts should be added incrementally here.
+
+module.exports = []
+
+
+/**
+ * Example:
+ * {
+    name: "Joel Coutinho",
+    title: "Backend Developer @Supertokens",
+    image: "https://avatars.githubusercontent.com/u/87567452?v=4",
+    socials: [
+      {
+        name: "github",
+        url: "https://github.com/jscyo",
+      },
+      {
+        name: "linkedin",
+        url: "https://github.com/jscyo",
+      },
+      {
+        name: "twitter",
+        url: "https://github.com/jscyo",
+      },
+    ],
+    description:
+      "The one major downside of TOTP is that the secret key is stored on both the user’s device and the server. If either of these systems were to be compromised, a malicious actor would now be able to generate codes and have unfettered access to the user’s account.",
+  },
```

Comment:
why is this in a js file? Each of the blog's md file has a front matter at the top of the file which already contains description title etc.. why have that here too?

Also, why can't we add author and socials to the front matter?

Reply:
- This js file maintains all the information regarding a blog author.
- yes we can add this information in each file in respective blogs but if we do that when we want to change something related to a author we need update it in every single md file. so this file acts as a single source of truth for blog author information.
- and `title` and `description`  are specific to blog in each md file. which is title of the blog and description of the blog. and here title refers to authors job title and his discription. btw changing `title -> jobRole`  and `description -> authorBio`